libcloud/test/compute/test_ec2.py

```diff
@@ -126,6 +126,14 @@ def test_create_node(self):
         self.assertEqual(node.extra['tags']['Name'], 'foo')
         self.assertEqual(len(node.extra['tags']), 1)
 
+        def test_create_node_with_launch_template(self):
```

Comment:
Overall the change looks good to me, but it would be good to add some additional tests which verify that the correct parameters are provided in the body when ``ex_launch_template_*`` keyword arguments are used.

This would need to happen inside the Mock response class.